src/test/java/racingcar/model/CarTest.java

```diff
@@ -0,0 +1,97 @@
+package racingcar.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import racingcar.exception.ExceptionMessages;
+import racingcar.util.generator.RandomNumberGenerator;
+
+public class CarTest {
+    private static Stream<Arguments> provideRandomNumbersForIsGreaterThanAndEqualFour() {
```

Comment:
```suggestion
    private static Stream<Arguments> provideRandomNumbersForIsGreaterThanOrEqualToFour() {
```
`>=`는 `greater than or eqal to`입니다.

Reply:
@jongmyeongbak 헉 감사합니다. 전 바본가봐요,, 🤣🤣